.travis.yml

```diff
@@ -5,10 +5,13 @@ cache:
   directories:
     - $HOME/.cache/pip
 
+postgres: 9.6
```

Comment:
don't needed here if it's already put below

Reply:
oh I see. thanks for the info